README.md

```diff
@@ -1,5 +1,8 @@
 # Mattermost Jenkins Plugin
 
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jenkins/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-jenkins)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jenkins/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-jenkins)
```

Comment:
`https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jenkins/master.svg` this says `invalid response data`

Reply:
I suppose this is because #16 isn't merged jet.

Reply:
Other PR is merged now, badge is present.